apps/sms/views/conversation/js/conversation.js

```diff
@@ -2403,6 +2415,10 @@ var ConversationView = {
   setMessageStatus: function(id, status) {
     var messageDOM = document.getElementById('message-' + id);
 
+    if (!messageDOM) {
```

Comment:
I'm not sure this fits in one line, but maybe on 2 lines it can be more readable indeed